plugins/wallabag.koplugin/main.lua

```diff
@@ -364,14 +372,24 @@ function Wallabag:download(article)
 end
 
 -- method: (mandatory) GET, POST, DELETE, PATCH, etc...
--- apiurl: (mandatory) excluding the server path
+-- apiurl: (mandatory) API call excluding the server path, or full URL to a file
 -- headers: defaults to auth if given nil value, provide all headers necessary if in use
 -- body: empty string if not needed
 -- filepath: downloads the file if provided, returns JSON otherwise
-function Wallabag:callAPI( method, apiurl, headers, body, filepath )
+function Wallabag:callAPI( method, apiurl, headers, body, filepath)
```

Comment:
Either do one or the other. ;-) (Our code prefers no spaces.)
```suggestion
function Wallabag:callAPI(method, apiurl, headers, body, filepath)
```